godot-core/src/builtin/aabb.rs

```diff
@@ -113,4 +428,379 @@ mod test {
 
         crate::builtin::test_utils::roundtrip(&aabb, expected_json);
     }
+
+    #[test]
+    fn test_axes_functions() {
+        let aabb = Aabb {
+            position: Vector3 {
+                x: 0.0,
+                y: 0.0,
+                z: 0.0,
+            },
+            size: Vector3 {
+                x: 4.0,
+                y: 6.0,
+                z: 8.0,
+            },
```

Comment:
You can use `Vector3::new()` here to use less vertical space.

Applies also everywhere below. Maybe a regex replace can do it, something like
```
Vector3\s*{\s*x:\s*([\d.]+),\s*y:\s*([\d.]+),\s*z:\s*([\d.]+),\s*}
```
replaced with
```
Vector3::new($1, $2, $3)
```
(untested)

Reply:
Thanks a lot for the RegEx mate! The final  one I used was: 
```regex
Vector3\s*{\s*x:\s*([-\d.]+),\s*y:\s*([-\d.]+),\s*z:\s*([-\d.]+),\s*}
```
To include negative numbers. Feel free to keep it in a corner!